noxfile.py

```diff
@@ -78,6 +91,27 @@ def test(session):
     session.run("pytest", *session.posargs)
 
 
+@nox.session(name="a11y")
+def a11y(session):
+    """Run the accessibility test suite."""
+    if _should_install(session):
+        session.install("-e", ".[test]")
+        # Install the drivers that Playwright needs to control the browsers
+        session.run("playwright", "install")
+    # Build the docs so we can run accessibility tests against them. If a base
+    # url is provided via the command line (see pytest-base-url plugin), then no
+    # need to build the docs, for example:
+    #
+    # ```console
+    # $ nox -s a11y -- --base-url="https://pydata-sphinx-theme.readthedocs.io/en/latest"
+    # ```
+    if not any(arg.startswith("--base-url") for arg in session.posargs):
+        session.run("nox", "-s", "docs", "--", "docs/", path_docs_build_for_a11y)
+    # The next step would be to open a server to the docs for Playwright, but
+    # that is done in the test file, along with the accessibility checks.
```

Comment:
I confess that I'm having trouble following the logic. I don't really understand why `--base-url` is needed.  I think I can see that it would allow you to run the test function on the remote, live site online (not just on a locally-built-and-served copy) but I'm not sure we actually need that functionality. Probably I'm just not understanding why it is necessary even for the locally-built case?